packages/pages/src/vite-plugin/build/buildStart/rendering/templateUtils.ts

```diff
@@ -97,7 +98,7 @@ const importRenderTemplate = async (
 export type GeneratedPage = {
   path: string;
   content?: string;
-  redirects: string[];
+  redirects: (RedirectSource | string)[];
```

Comment:
You need to update the redirects field in the return of generateResponses and map them to the new type. For any _just string_ items in our array we need to convert to the new object with a default status of 301.

Reply:
Based on the comment here: https://yext.slack.com/archives/C06PKRZN4AW/p1712170611016879?thread_ts=1712167837.746649&cid=C06PKRZN4AW

We no longer want to do this and want to keep passing through the redirects as is?